frappe/patches/v11_0/remove_skip_for_doctype.py

```diff
@@ -60,7 +60,7 @@ def execute():
 					# Maintain sequence (name, user, allow, for_value, applicable_for, apply_to_all_doctypes, creation, modified)
 					new_user_permissions_list.append(
 						(
-							frappe.generate_hash("", 10),
+							frappe.generate_hash(lenght=10),
```

Comment:
Typo (also elsewhere in the PR, but I'll just note this one)

Reply:
Fixed, thanks!